tour/getting-started.md

```diff
@@ -2,4 +2,29 @@
 title: Getting Started
 ---
 
-Talk about how to get started.
+First make sure you have Java, Maven and Git installed on your machine.  Oh you are already rocking? OK let's go!
+
+1. Clone the tour onto your machine:
+```commandline
+git clone -b tour https://github.com/apache/accumulo-website.git tour
+cd tour
+```
+2. Open Main.java in your favorite editor.
```

Comment:
Could make Main.java link to https://github.com/apache/accumulo-website/blob/tour/src/main/java/tour/Main.java